src/components/App.tsx

```diff
@@ -0,0 +1,15 @@
+import { RouterProvider } from 'react-router-dom';
+import CssBaseline from '@mui/material/CssBaseline';
+import Container from '@mui/material/Container';
+import router from '../routes/routes.tsx';
+
+export default function App() {
+    return (
+        <>
+            <CssBaseline />
+            <Container maxWidth={false} sx={{maxWidth: "2560px"}}>
+                <RouterProvider router={router} />
+            </Container>
+        </>
+    )
+}
```

Comment:
@DamianStrz Please update the indentation in this file to use 2 spaces for tabs instead of 4. It looks like the current indentation is using 4 spaces, but to maintain consistency with the project's coding standards, it should be 2

Reply:
Please use an arrow function for the App component. To maintain consistency across the project, let's stick to using arrow functions for all functional components moving forward.